Add the volunteer perks and recruiting form (#612)
EuroPython/website: src/content/pages/volunteers.mdx

subtitle: We need your help to make EuroPython 2024 a success!
---

> # Volunteering @ EuroPython

Reviewer: Why? Is this just a preference or is there something wrong on the page?

Author: Just a preference, not a strong one though. We can keep it as is

Reviewer: It seems we have no precendence for 2024 ... - So let's go with your version for now.

Python owes much of its success to open source and the community surrounding it. In the same spirit, EuroPython is organized and run by volunteers from the Python community. Alas, we're only a few and need your help to make the show run as smoothly as possible. Each year we are joined by our awesome attendees as volunteers to help in running this wonderful conference. This page explains some of the ways you can help us.

Being a volunteer means giving your time for free, without expecting anything back, just for the pleasure of being a part of this amazing conference. It's a great way to meet new people, learn new things, and have a lot of fun. You can volunteer for a few hours or for the whole conference, it's up to you.

We will also have a small number of perks for you, some small gifts to say "thank you" for your time and for your help.

> 

Reviewer: This does not seem to make sense here.

# Things to remember

* This is **in-person** volunteering and would require you to be present at the venue.
* While we strive to make the conference accessible to everyone, we cannot provide travel or accommodation support to volunteers.
* To enter the conference you need a ticket.

> * You are welcome to volunteer as much or as little as you want to. We appreciate any help you can give us. However please note we will not be able to refund you for the ticket, should you choose to volunteer for ten (10) hours or more. We are unable to commit to that at this point.

Reviewer: This is not correct. We can refund tickets if people choose to switch over to a volunteer ticket.

Author: Got this one from the last year's website. It says:

> You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket. However please note we will not be able to refund you for the ticket should you choose to volunteer for eight hours or more. We are unable to commit to that at this point.

Reviewer: We can **not** do it **after** the event. 
But we can do it before. Or better they can cancel their ticket like everybody else before the event (preferably after they got a voucher).

Author: Thanks ❤️

# Things to remember

* This is **in-person** volunteering and would require you to be present at the venue.
* While we strive to make the conference accessible to everyone, we cannot provide travel or accommodation support to volunteers.
* To enter the conference you need a ticket.
* You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket. We appreciate any help you can give us.

# Perks for volunteers

> * 🎟️ A free combined ticket (Tutorials + Conference) if you volunteer for ten (10) hours or more

Reviewer: ```suggestion
* 🎟️ The possibility of a free combined ticket (Tutorials + Conference) if you volunteer for ten (10) hours or more
```
Idea is to make it sound a bit less like a done deal.

# Things to remember

* This is **in-person** volunteering and would require you to be present at the venue.
* While we strive to make the conference accessible to everyone, we cannot provide travel or accommodation support to volunteers.

> * To enter the conference you need a ticket. Some volunteers are eligible for a free ticket, see perks below.

Reviewer: ```suggestion
* To enter the conference you need a ticket.
```
I do not want to have that much of a focus on this. And with this one it will be mentioned three times.

* You are welcome to volunteer as much or as little as you want to, even if you already hold a ticket. We appreciate any help you can give us.

> * You can get a free combined ticket (Tutorials + Conference) if you pledge to volunteer for ten (10) hours or more.

Reviewer: ```suggestion
```
As above. I don't want to announce this too much. It is a perk if you find it. And you will be asked at a later stage anyway.
This same sentence is repeated 3 lines later. Sounds like we are trying to get people to go for the tickets.

# Perks for volunteers

> * A free combined ticket (Tutorials + Conference) if you volunteer for ten (10) hours or more!

Reviewer: Maybe even put this at the end of the list. - But that is just my current preference at this moment.

* Handing out registration badges
* Checking student IDs
* Answering general questions

We want to call out that volunteers at the registration desk will generally not be able to attend talks for the duration of the task.

## T-Shirt Handout

* Handing out the conference T-shirts to attendees
* This job involves standing and lifting light items

We want to call out that this job is located at the registration desk, and you will generally not be able to attend talks for the duration of the task.

## Room Manager

A room manager is responsible for ensuring everything in a room runs smoothly for the entirety of a slot. Some of the activities a room manager is expected to carry out involve:

> * Ensuring people are seated before a talk, and the room is not overcrowded

Author: ```suggestion
* Ensuring that all available space is utilized, people are seated before the talk begins, and the room is not overcrowded.
```

Author: @cmaureir I think we can phrase it like this

* Making sure talks start and finish on time
* Moderating Q&A sessions, assisting with passing the microphones and reading out questions from remote attendees

## Setup/Teardown

* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare, etc.)
* On the last day (Friday afternoon for the main conference, Sunday for the sprints), there will be a venue cleanup afterwards
* This job involves walking and carrying items, some of which may be heavy

## Greeter / Badge Check

* Checking attendees at the venue for badges
* Sending people who do not have a badge to the registration desk, unless they are part of the venue staff (e.g., AV team, catering, etc.)
* Reporting people who don't have badges and refuse to go to the registration desk

## Runners and Ad-hoc Activities

There are always some tasks which we simply cannot plan beforehand. Based on what's needed in the moment, you can be expected to do one or more of the following (not an exhaustive list):

* Checking rooms to ensure all the necessary resources are in place
* Helping out with Registration Desk, T-shirt Handout, or Room Manager duties
* Moving chairs, tables, or poster walls

> * Arranging transfers between venue and airport, etc. (rarely)

Author: @artcz

> * Arranging transfers between venue and airport, etc. (rarely)

Reviewer 2: As discussed, IMO we could remove this item. I would direct people who need this kind of help to the debugging desk.

> * Arranging transfers between venue and airport, etc. (rarely)

Author: ```suggestion
```

* This job involves standing and lifting light items

We want to call out that this job is located at the registration desk, and you will generally not be able to attend talks for the duration of the task.

## Room Manager

A room manager is responsible for ensuring everything in a room runs smoothly for the entirety of a slot. Some of the activities a room manager is expected to carry out involve:

* Ensuring people are seated before a talk, and the room is not overcrowded
* Making sure the session chair and speakers are in the room on time
* Keeping in touch with the AV team to ensure everything is in order
* Keeping the [Code of Conduct (CoC)](https://www.europython-society.org/coc/) in mind and contact CoC committee members if needed

We want to call out that room managers might not be able to attend or focus on talks for the duration of the task.

## Session Chairing

A session chair plays a key role in introducing talks and ensuring a seamless transition between talks scheduled during a slot. The activities that a session chair is expected to carry out include:

* Helping speakers prepare their computers for the talks
* Introducing the speakers to the audience

> * Signalling the speakers how much time is left

Reviewer: Don't think this change makes sense.

* Making sure talks start and finish on time
* Moderating Q&A sessions, assisting with passing the microphones and reading out questions from remote attendees

## Setup/Teardown

* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare, etc.)
* On the last day (Friday afternoon for the main conference, Sunday for the sprints), there will be a venue cleanup afterwards
* This job involves walking and carrying items, some of which may be heavy

## Greeter / Badge Check

* Checking attendees at the venue for badges

> * Sending people to the registration desk if they don't have a badge, and not from the venue (e.g. AV team, catering, etc.)

Reviewer: This sentence does not work. It does no longer express that people from the venue are excempt from the badges.

* Reporting people who don't have badges and refuse to go to the registration desk

## Runners and Ad-hoc Activities

There are always some tasks which we simply cannot plan beforehand. Based on what's needed in the moment, you can be expected to do one or more of the following (not an exhaustive list):

> * Checking the rooms to ensure all the necessary resources are in place

Reviewer: If we add "the", is should be "Check"

Author: The presence of "the" doesn't change the verb from "checking" to "check". Both "checking the rooms" and "checking rooms" are correct, but you're right. We shouldn't use "the" because we haven't mentioned any specific rooms to be checked.

* Helping out with Registration Desk, T-shirt Handout, or Room Manager duties
* Moving chairs, tables, or poster walls
* Arranging transfers between venue and airport, etc. (rarely)

> 

Reviewer: Unneccesary \n

# How to sign up

> After you fill out the form below and acknowledge our [Data Privacy Policy](https://forms.gle/PCP7bXvdH1QhDBPn6), we will reach out to you with a link to join the EuroPython Discord server.

Reviewer: This is not correct. The request for Data Privacy Policy is part of the form in the middle of the next paragraph. But filling it in will not lead to a discord invite.
And we will not invite people right away. There will be a selection based on the next form.

That said the form needs to be filled in.

Reviewer: Sounds better.

> Please say hello and introduce yourself in the *#onsite-volunteers* channel.

Reviewer: ```suggestion
```

> Thanks ❤️

Reviewer: ```suggestion
```

<div class="text-center mb-8">
<ButtonLink href="https://forms.gle/MV4xuRkHq5eN8efZ9">Register your interest now!</ButtonLink>
</div>

> 

Reviewer: ```suggestion
```
